packages/mdc-ripple/_mixins.scss

```diff
@@ -101,6 +109,32 @@
   }
   // stylelint-enable at-rule-empty-line-before, block-closing-brace-newline-after
 
+  &.mdc-ripple-upgraded {
+    &#{$pseudo} {
+      top: calc(50% - #{$radius});
+      left: calc(50% - #{$radius});
+      width: $radius * 2;
+      height: $radius * 2;
+      transform: scale(0); // fallback for browsers that don't support custom properties
```

Comment:
You can remove the inline comment; this is a convention in our codebase which we talk about in our mdc-theme package.